Classes/Extension/ExtensionSetupRenderer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace Helhum\Typo3Console\Extension;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Database\Schema\SchemaUpdateResult;
+use Helhum\Typo3Console\Database\Schema\SchemaUpdateType;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\SignalSlot\Dispatcher;
+use TYPO3\CMS\Extensionmanager\Utility\InstallUtility;
+
+/**
+ * Class ExtensionSetup
```

Comment:
wrong description. completely remove it or add something useful to understand what this class does